modules/selenium/src/test/java/org/testcontainers/junit/BaseWebDriverContainerTest.java

```diff
@@ -1,62 +1,62 @@
 
 import static java.lang.String.format;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
 import static org.rnorth.visibleassertions.VisibleAssertions.assertTrue;
 
 /**
  *
  */
 public class BaseWebDriverContainerTest {
 
-    protected void doSimpleWebdriverTest(BrowserWebDriverContainer rule) {
+    @ClassRule
+    public static Network NETWORK = Network.newNetwork();
+
+    @ClassRule
+    public static GenericContainer<?> HELLO_WORLD = new GenericContainer<>(DockerImageName.parse("testcontainers/helloworld:1.0.0"))
+        .withNetwork(NETWORK)
+        .withNetworkAliases("helloworld")
+        .withExposedPorts(8080, 8081)
+        .waitingFor(new HttpWaitStrategy());
+
+    protected static void doSimpleExplore(BrowserWebDriverContainer<?> rule) {
         RemoteWebDriver driver = setupDriverFromRule(rule);
         System.out.println("Selenium remote URL is: " + rule.getSeleniumAddress());
         System.out.println("VNC URL is: " + rule.getVncAddress());
 
-        driver.get("http://www.google.com");
-        WebElement search = driver.findElement(By.name("q"));
-        search.sendKeys("testcontainers");
-        search.submit();
+        driver.get("http://helloworld:8080");
+        WebElement title = driver.findElement(By.tagName("h1"));
 
-        List<WebElement> results = new WebDriverWait(driver, 15)
-            .until(ExpectedConditions.visibilityOfAllElementsLocatedBy(By.cssSelector("#search h3")));
-
-        assertTrue("the word 'testcontainers' appears in search results",
-            results.stream()
-                .anyMatch(el -> el.getText().contains("testcontainers")));
+        assertEquals("the index page contains the title 'Hello world'",
+            "Hello world",
+            title.getText().trim()
+        );
     }
 
-    protected void assertBrowserNameIs(BrowserWebDriverContainer rule, String expectedName) {
+    protected void assertBrowserNameIs(BrowserWebDriverContainer<?> rule, String expectedName) {
         RemoteWebDriver driver = setupDriverFromRule(rule);
         String actual = driver.getCapabilities().getBrowserName();
         assertTrue(format("actual browser name is %s", actual),
             actual.equals(expectedName));
     }
 
     @NotNull
-    private static RemoteWebDriver setupDriverFromRule(BrowserWebDriverContainer rule) {
+    private static RemoteWebDriver setupDriverFromRule(BrowserWebDriverContainer<?> rule) {
         RemoteWebDriver driver = rule.getWebDriver();
         driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
         return driver;
     }
-
-    protected static void doSimpleExplore(BrowserWebDriverContainer rule) {
-        RemoteWebDriver driver = setupDriverFromRule(rule);
-        driver.get("http://en.wikipedia.org/wiki/Randomness");
-
-        // Oh! The irony!
-        assertTrue("Randomness' description has the word 'pattern'", driver.findElementByPartialLinkText("pattern").isDisplayed());
-    }
```

Comment:
There's no real sense in having two ways to test that Selenium is working, so just removed this altogether.